bin/metadata_validation.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python3
+
+import sys
+import re
+
+import json
+
+import synapseclient
+
+
+def get_synaspe_file(syn_id: str) -> synapseclient.entity.File:
```

Comment:
@BWMac
Nit: there is a typo here.
Nit: Add in the docstring the argument parameters:

```
"""
description...

Args:
    syn_id (str): Synapse Id
...
```

I would recommend using the vscode extension (audoDocstring - Python docstring generator).  It takes the type hints you have and auto generates docstrings with a nice format.